hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -293,8 +302,6 @@ def _step_wiping(self, verbose: bool) -> StepRes:
              [Status.BUSY_SEC_WIPE_INT, Status.LOCKED])):
             self.state.ext_regs.write('STATUS', Status.BUSY_SEC_WIPE_INT, True)
 
-        is_good = self.state.get_fsm_state() == FsmState.WIPING_GOOD
-
```

Comment:
I think this should not be removed, for the reason given in the previous comment.